simpleclient_log4j/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prometheus</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.0.9-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.prometheus</groupId>
+    <artifactId>simpleclient_log4j</artifactId>
+    <packaging>bundle</packaging>
+
+    <name>Prometheus Java Simpleclient log4j</name>
+    <description>
+        Metrics collector for log4j appender logging
+    </description>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>willfleury</id>
+            <name>Will Fleury</name>
+            <email>will.fleury@boxever.com</email>
+        </developer>
+    </developers>
+
+    <properties>
+        <log4j.version>1.2.17</log4j.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
not if the person was including this dependency only (i.e. the only wanted to use the log metrics and nothing else).. 

The user shouldn't be including different versions of this module and the core anyway.